src/redux/actions/genes/loadDownsampledGeneExpression.js

```diff
@@ -0,0 +1,138 @@
+import _ from 'lodash';
+import { SparseMatrix } from 'mathjs';
+
+import {
+  DOWNSAMPLED_GENES_LOADING,
+  DOWNSAMPLED_GENES_LOADED,
+  DOWNSAMPLED_GENES_ERROR,
+} from 'redux/actionTypes/genes';
+
+import fetchWork from 'utils/work/fetchWork';
+import getTimeoutForWorkerTask from 'utils/getTimeoutForWorkerTask';
+
+// Debounce so that we only fetch once the settings are done being set up
+const loadDownsampledGeneExpressionDebounced = _.debounce(
+  async (
+    experimentId,
+    genes,
+    componentUuid,
+    dispatch,
+    getState,
+  ) => {
+    const state = getState();
+
+    const {
+      groupedTracks,
+      selectedCellSet,
+      selectedPoints,
```

Comment:
what is `selectedPoints`?

Reply:
One of the settings that the plots and tables heatmap can modify, it conditions what cells can be shown